man/git-brv.md

```diff
@@ -3,12 +3,18 @@ git-brv(1) -- List branches sorted by their last commit date
 
 ## SYNOPSIS
 
-`git-brv`
+`git-brv` [-r|--reverse]
 
 ## DESCRIPTION
 
   Pretty listing of branches sorted by the date of their last commit.
 
+## OPTIONS
+
+  -r | --reverse
+
+  Reverses the output to put the most recent branch at the bottom of the list. This is useful when there are large amount of branches as the most recent branch is shown just above the next prompt. This can be configured as the default by setting git-extras.brv.reverse to true in your git options.
```

Comment:
```suggestion
  Reverses the output to put the most recent branch at the bottom of the list. This is useful when there are large amount of branches as the most recent branch is shown just above the next prompt. This can be configured as the default by setting `git-extras.brv.reverse` to true in your git options.
```
This helps to generate a better HTML page.